app/src/main/java/nyc/c4q/jonathancolon/inContaq/utlities/sms/Sms.java

```diff
@@ -5,57 +5,57 @@
 @Parcel
 public class Sms implements Comparable<Sms>{
     public String _id;
-    public String _address;
-    public String _msg;
-    public String _readState;
-    public String _time;
-    public String _folderName;
-    public String _type;
+    public String address;
```

Comment:
either make them private and keep the getters and setters, or keep them public and remove the getters and setters